server/channels/api4/handlers.go

```diff
@@ -18,9 +18,15 @@ type Context = web.Context
 
 type handlerFunc func(*Context, http.ResponseWriter, *http.Request)
 
+type APIHandlerOption string
+
+const (
+	handlerParamFileAPI = APIHandlerOption("fileAPI")
```

Comment:
I like better this approach. Nevertheless, I feel the name could be better. Maybe something like "useFileSizeLimit" is more revealing?

If you want to do it more agnostic, maybe something like "expectFile"?